Sources/Client/Views/Styles/Colors.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+import SwiftUI
+
+enum Colors {
+  case primaryDarkGray, secondaryDarkGray
+  case warningYellow
+  
+  var color: Color {
+    switch self {
+    case .primaryDarkGray: return Color(red: 32/255, green: 32/255, blue: 32/255)
+    case .secondaryDarkGray: return Color(red: 83/255, green: 83/255, blue: 83/255)
+    case .warningYellow: return Color(red: 231/255, green: 241/255, blue: 89/255)
+    }
+  }
+}
```

Comment:
I think this would be better. I don't think there's a reason they need to be enum cases. I've left it as an `enum` because for swift that's how we do constants namespaces (basically the same as a struct with `private init() {}`.

```suggestion
enum Colors {
  static var primaryDarkGray = Color(red: 32/255, green: 32/255, blue: 32/255)
  static var secondaryDarkGray = Color(red: 83/255, green: 83/255, blue: 83/255)
  static var warningYellow = Color(red: 231/255, green: 241/255, blue: 89/255)
}
```